sdk-utils/src/main/java/com/redhat/parodos/sdkutils/SdkUtils.java

```diff
@@ -57,7 +51,7 @@ public static ApiClient getParodosAPiClient()
 		}
 
 		int port = Integer.parseInt(serverPort);
-		if (port <= 0 && port > 65535) {
+		if (port <= 0 || port > 65535) {
```

Comment:
this isn't related to this PR, but fixes a bug since this if used to return always false.

Reply:
you're right. Fixes even also https://github.com/parodos-dev/parodos/pull/376/commits/e837ae60a4b119f394791728aa4f3c96dffcf3cf